lib/usdUfe/ufe/UsdClipboard.cpp

```diff
@@ -116,4 +161,15 @@ void UsdClipboard::cleanClipboardStageCache()
 
 void UsdClipboard::removeClipboardFile() { std::filesystem::remove(_clipboardFilePath); }
 
+void UsdClipboard::ufeSelectionChanged()
+{
+    if (!_inSelectionGuard) {
+        // Any ufe selection changed event that occurs between the copy and paste (that didn't
+        // come from the paste itself) means that the user has selected what they want to be
+        // the paste target.
+        _pasteAsSibling = false;
+        _ufeSelObserver.reset();
+    }
+}
```

Comment:
The selection guard is set from the paste command since it actually modifies the selection by selecting the newly pasted prims. We want to ignore that selection. But any other selection will remove the observer and reset the paste as sibling.